README.md

```diff
@@ -23,6 +23,13 @@ And appropriately set the following environment variables:
 - `$SLACK_CHANNEL_ID`: the channel ID number.
 - `$SLACK_TIMBOT_USER_ID`: the timbot slack user ID.
 
+To use the database, you'll have to set the following environment variables as well:
+
+- `$SQL_HOST`: Your MySQL host
```

Comment:
<Your MySQL host>

Reply:
nah

Reply:
this isn't how the other vars in the README are formatted

Reply:
@timflannagan1 @Daniel-Pivonka which direction do we want to go i, `<>` or no `<>`?

Reply:
`<>`